src/taoensso/sente/server_adapters/undertow.clj

```diff
@@ -13,6 +13,8 @@
      WebSocketConnectionCallback
      WebSocketProtocolHandshakeHandler]))
 
+(def ajax-response-timeout-ms (* 60 1000))
```

Comment:
Would suggest instead adding this as an option to `get-sch-adapter`

Reply:
Relatedly: please add a docstring to `get-sch-adapter` to document the 2x new options:

- `:ajax-response-timeout-ms` (and mention its default)
- `:ajax-response-timeout-val` (and mention its default)

`get-sch-adapter` should accept zero args (use default opts), or a single map arg of opts.

Ideally the PR would have 2x commits:
- 1st commit to introduce the new options, but keep the old default behaviour (non-breaking).
- 2nd commit to change the default, which we can mark as potentially breaking.